### ISSUE TITLE
[BP-BUG] Stars and Crime level not showing in Logs

### ISSUE THREAD

**Issue by Qcoomber** [PointLife/PointLife-Issues#4]
**Describe the bug**
The title is pretty self-explanatory. Both the logs in the Pointlife.net staff panel and the new OpenPPC (Good shi) will not show the player's crime/star despite the fact that the log shows that it's supposed to be there (imagine teasing).

**To Reproduce**
1. None

**Expected behavior**
The logs to show the amount of crime/star of a specific player.

**Screenshots**

**Additional context**
This problem has existed since the bane of existence (not really) and I just want to stop players from crying RDA all the time and complaining about us not being able to get the proof (mostly their fault + skill issue).

**Comment by PLASMAchicken** [comment]
Yeah this was broken with the GameSource into LifeSource seperation. Will fix.
